packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/assureEmojis.st

```diff
@@ -0,0 +1,9 @@
+downloading
+assureEmojis
```

Comment:
They probably named it assure instead of ensure because there is a method called FileDirectory>>assureExistence. In my opinion we should rename it to ensure

Reply:
Moved `assureExistence` to `downloadArchive` as that is the only time it is actually needed. This should also increase performance.